### PR TITLE
[chore] updated README, installed tailwindcss, deleted setup scripts

### PR DESCRIPTION
**Pull request by ethan-tam33**
[//]: # "Feel free to customize this template and the emojis to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- deleted PL todos from README
- installing tailwindcss for all styling
- deleting all setup scripts

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## 👍 How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## 👍 Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @ethan-tam33

### CODE REVIEW

File: package.json
[the remark this answers is not in the document]
Comment (ethan-tam33) [reply package.json:1]: what needs to be changed in package.json tho?

[the remark this answers is not in the document]
Comment (jinkang-0) [reply package.json:1]: oh i meant the name field, it says "web app template" rn still

File: actions/supabase/client.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { createClient } from "@supabase/supabase-js";
+import { createClient } from '@supabase/supabase-js';
```
Comment (saitunguturu) [inline actions/supabase/client.ts:1]: Why did you change it from double to single quotes? Personal preference or functionality difference?

**Review by saitunguturu — APPROVED** [review]

**Review by jinkang-0 — APPROVED** [review]
lgtm

File: actions/supabase/client.ts (hunk 1 above)
Comment (jinkang-0) [reply actions/supabase/client.ts:1]: image
It's likely a change as a result of the change to the Prettier config that standardizes single vs double quotes. Usually these are just personal preferences, but it's "prettier" to have it standardized across the app. Of course, sometimes it's functionally significant to have a string defined as something like `"Abigail's Flower"`, where double quotes are preferred so it's easier to read the single quote inside without having to escape (e.g. as opposed to `'Abigail\'s Flower'`), but I digress.
